_posts/2024-05-31-friends-dont-let-friends-package-alone-pyconus-24.md

```diff
@@ -0,0 +1,282 @@
+
+Among other things, pyOpenSci reviews software <link> as a constructive way to help scientists both create better software and get credit for the important work they are doing.
+
+The pyOpenSci community has created a Python package guide that harnesses the expertise, opinions, and experiences of 55 people. This guide provides:
+
+* a beginner-friendly overview of the packaging ecosystem combined with a
+* start-to-finish tutorial that uses Hatch <link> to create a Python package.
+
+Python is known for its incredible community, so it only made sense to leverage that collective knowledge and experience to create a packaging guide. Packaging doesn’t have to be complex. Friends can help.
+
+The talk can be broken down into 4 takeaways:
+
+
+### People can help (_surprise_!)
+
+When tackling some of the most thorny technical Python problems, people can help. This might mean:
+
+* Learning together by working in a small supportive group to create your first Python package;
+* Bringing in experts to review and vet online resources to ensure they are accurate and accessible that support scientists worldwide in their packaging journey.
+* Running a constructive peer review process where people volunteer their time to provide feedback on a package’s usability, function, and structure.
+
+In creating the guide, we knew we were tackling hard, complex technical problems. But also there is a tremendous amount of collective knowledge in this community that can be harnessed and focused on the problem at hand. By focusing people on the users of the guide first, rather than tools, we were able to build consensus.
+
+We were able to move forward together.
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/june/pyopensci-python-packaging-people-can-help.png" alt="Image on a dark purple background that says - people can help. below are two people stick figure like loking at each other and smiling while chatting." style="width: 70%;" />
+  <figcaption><i class="fa-solid fa-heart-pulse"></i> When tackling Python's thorniest challenges, people can help.  <i class="fa-solid fa-heart-pulse"></i>.</figcaption>
+</figure>
+
+### Create early wins for users
+
+As an educator, I’ve always known that the best way to empower someone new to any topic is to create a learning environment where they quickly make progress. I’ve [collected data on what approaches work better than others](https://osf.io/preprints/osf/xdj4z). I’ve explored how certain presentations of information empower, while others break people down triggering imposter syndrome and even isolation within a science field.
+
+The key is early wins.
+
+Early wins empower users to:
+
+* Want to do more
+* Feel like they belong and can be successful learning the skill at hand
+* Try again and expand their knowledge
+* Even help their colleagues which is the biggest win of all. \
```

Comment:
```suggestion
* Even help their colleagues, which is the biggest win of all. 
```

Reply:
I don't know if I would personally put helping my colleagues above a sense of belonging!

Reply:
oh interesting. i definitely have found that when you run mentorship programs and or any sort of learning program / envt, the most empowering thing for a student is when you bring them back or they can get to a state where they are a  mentor  / able to help others. Because

1. it builds even more confidence 
2. you learn a ton as a mentor / helping others --> more confidence / more belonging and
3. when a new mentee / new learner sees someone "like them" that has moved into a mentorship role or a role where that person is helping others they then see themselves as belonging even more!!

the same is true for a classroom. when a student realizes they can help someone else it's really a cool confidence building situation. 

That is when i know the class has been successful - they no longer depend on the teacher /the online resources / etc. they are empowered!

i actually studied this - specifically with tribal and other underserved schools. it's powerful.